src/com/sbezboro/standardplugin/listeners/CreatureSpawnListener.java

```diff
@@ -45,8 +47,7 @@ public void onCreatureSpawn(CreatureSpawnEvent event) {
 			}
 		}
 
-		// Temprorarily disable
-		if (false && CONTROLLED_ENTITIES.contains(entity.getType())) {
```

Comment:
I "Temprorarily" disabled this back in 2014... I don't remember why, but presumably for performance reasons? I Think iterating through every entity in a chunk may be a bit costly

Reply:
will do some performance tests on this